Python/PRP/pyCreatableConvert.cpp

```diff
@@ -232,6 +239,7 @@
 #include "PRP/Modifier/pyMaintainersMarkerModifier.h"
 #include "PRP/Modifier/pyModifier.h"
 #include "PRP/Modifier/pyOneShotMod.h"
+#include "PRP/Modifier/pyPostEffectMod.h"
 #include "PRP/Modifier/pyPythonFileMod.h"
 #include "PRP/Modifier/pyResponderModifier.h"
 #include "PRP/Modifier/pySpawnModifier.h"
```

Comment:
IConvert had the classes commented out, but ICreate never had the whole set (well, not since 2010).  You may have had them in the first set, but missed the second, which would also explain why now there is nothing.

I'm not sure the current order is consistent or meaningful, since I see commits which add both to the middle and to the end.  Only non-abstract types need to be in the list, since it should only take real constructed objects to upcast.  However, it should also be safe (and easier) to put all creatable types in the list, so I'd probably lean towards that option.